edudoc/judge-tutorial/judge-tutorial.md

```diff
@@ -45,7 +45,7 @@ Here are some common incidents that can result in +2 or DNF:
 
 **+2**: the competitor stops the timer without fully releasing the puzzle.
 
-**DNF**: the competitor touches the puzzle before the judge has inspected the solved state. If you are certain that no change in alignment of the puzzle was done as a result of this, **+2** can be given instead.
+**DNF**: the competitor touches the puzzle before the judge has inspected the solved state. If you are certain that no change in alignment of the puzzle was done as a result of this, **+2** can be given instead. Exception: if immediately after stopping the timer, the competitor **briefly** touches the puzzle without affecting its state, the time penalty may be lifted, at the discretion of the judge

```

Comment:
Missing period at the end. Also, is it not at the discretion of the Delegate?

Reply:
Nope, [A6c](https://www.worldcubeassociation.org/regulations/#A6c). Added period.